haml/_header.html.haml

```diff
@@ -1,9 +1,18 @@
-- links ||= []
-- sign_in_out ||= {}
+:ruby
```

Comment:
How come you are using the `:ruby` filter rather than sticking with `-` for each line?

Reply:
It's a haml-linting rule that error if there are three or more